app/src/main/AndroidManifest.xml

```diff
@@ -15,13 +15,12 @@
         android:theme="@style/Theme.QuizAcademyTask">
         <activity
             android:name=".MainActivity"
-            android:configChanges="orientation|screenSize"
+            android:configChanges="screenSize"
```

Comment:
why do you need to handle screenSize?

Reply:
Das war ein paste für übliche configchanges handling, wird aber nicht mehr gebraucht, kann ich entfernen